src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -106,6 +106,32 @@ public Map<Name, BigFraction> getAllPortions() {
             );
     }
 
+
+    /**
+     * Returns the portion of the transaction that the person has to pay the user.
```

Comment:
Can add a comment to Javadocs to explain when this function returns negative amount here?